ofrak_core/test_ofrak/components/test_free_space.py

```diff
@@ -63,11 +63,12 @@ async def test_partial_free_modifier(resource_under_test: Resource):
     """
     Test that the PartialFreeSpaceModifier returns expected results.
     """
-    partial_start_offset = 4
-    partial_end_offset = 10
-    parent = await resource_under_test.get_parent()
+    partial_start_offset = 0x14
+    partial_end_offset = 0x1A
```

Comment:
I don't believe this update is needed.

This offset _should_ be the offset in the memory region, not in the parent resource, so 4 and 10 should still work here.

What this test should be doing is replacing bytes 4-10 with the stub (which is not free space) plus `\x00` * remaining length of the modification, which would be 2 bytes in this case. Do you think we can refactor it to make this intention clearer?

Reply:
These numbers shouldn't be offsets at all, they're addresses in the `MemoryRegion` resource. I've updated the variable names to make this clearer and changed the next line to make sense.

I've also changed the `MemoryRegion` virtual address to `0x100` so we don't run into more bugs hidden by only testing with `0`